public/js/components/Filters/Filters.js

```diff
@@ -1,100 +1,109 @@
 import React from 'react';
-import {Grid, Row, Col} from 'react-flexbox-grid';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import moment from 'moment';
+import { DateRangePicker } from 'react-dates';
 
-const Filters = ({onSelectChange, filterVals, updating}) => {
-    return (
-        <form className="form">
-            <Grid fluid>
-                <Row>
-                    <Col xs={12} md={4}>
-                        <div className="form__row">
-                            <label>
-                                Filter by Office:
-                                <select
-                                    className="form__field form__field--select"
-                                    onChange={event => onSelectChange({
-                                        office: event.target.value
-                                    })}
-                                    value={filterVals.office}
-                                    disabled={updating}
-                                >
-                                    <option value="all">All</option>
-                                    <option value="uk">UK</option>
-                                    <option value="au">AU</option>
-                                    <option value="us">US</option>
-                                </select>
-                            </label>
-                        </div>
-                    </Col>
+class Filters extends React.Component {
+    constructor(props) {
```

Comment:
You can avoid these constructors with the `babel-plugin-transform-object-assign` package.